meeting-notes/2018/2018-10-22--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,140 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 October 22, 2018
+
+- **Lead:** @alanshaw
+- **Notetaker:** @jacobheun
+- **Attendees:**
+  - @terichadbourne
+  - @vasco-santos
+  - @victorbjelkholm
+  - @hugomrdias
+  - @vmx
+- **Recording:** TBC
```

Comment:
Where is the mighty recording?

Reply:
Recording added and the video has been properly ordered in the js core playlist on youtube as well.